### ISSUE TITLE
Bug: "litexa deploy" fails to work with message "Cannot read property 'name' of undefined"

### ISSUE THREAD

**Issue by fboerncke** [alexa-games/litexa#99]
# Bug Report

I really appreciate your work regarding Litexa and I would like to try some ideas which come up when I read your documentation. Unfortunately I have problems to get your examples up and running.

## Litexa Version

0.3.1

## Scenario

Here is my scenario:

- I am located in Germany and I would like to deploy the skill to region "eu-west-1" using "de-DE". I do not find any documentation to make this work.
- I already have an up and running ask-cli and aws-cli which I used for skill development in the past using the "default" profile. I attached the suggested IAM policy to the "default" profile.

## Current Behavior

When when I call the command "litexa deploy" whatever I try I keep getting the following message:

[deploy] +3ms Cannot read property 'name' of undefined

## Expected Behavior

I would expect to have the artefacts deployed to my account which doesn't happen.

Now my questions:

(1) Is it possible to use the command "litexa deploy" together with some command line option to make it more verbose in case of errors?
(2) Did you ever successfully deploy to another region/locale than US?
(3) Maybe you have some work in progress documentation for such scenario which you would like to share with me?

## Operating System

Linux Mint

## Node Version

v8.10.0

Thanks for your efforts regarding this project!

**Comment by ayrc** [comment]
Hi Frank,

Thanks for writing this issue!

Regarding your first ask on deploying to 'eu-west-1' as default: We don't have this as a feature yet. It is currently being tracked in #77. In the meantime, you might be able to set regional endpoints in your skill manifest by writing it in skill.js (untested) or manually changing it in the Developer Console (but you'd have to do this each time you deploy your skill manifest). Additionally, is your AWS profile's region configured to 'eu-west-1'? I'm trying to see if that is a factor Litexa takes into account when deploying - this is something we haven't tested yet.

As for your `[deploy] +3ms Cannot read property 'name' of undefined` issue, this is usually a parsing error from reading the Litexa config file. Can you type `node litexa.config.js` into your command line and see if a syntax error shows up? Surfacing the specific error is something we can make an action item of this issue.

Now, regarding your questions:
1) All `litexa` commands accept a verbose logging flag `--verbose`. That should give you more information.
2) Not yet; this is tracked in issue #77.
3) We don't document work in progress issues explicitly - you can look at issues individually to track their progress, and you can also look at the `develop` branch to see what features will be released in the future.

**Comment by fboerncke** [comment]
Hi Alina, thanks for your fast feedback which turned out to be helpful.

The hint regarding "--verbose" and "node litexa.config.js" gave me so much insight into what was going wrong that I could find my own error which was related to missing quote characters ==> my fault. 

Furthermore I added some entries in "skill.js" as suggested.

Within the file "lambda.coffee" in "@litexa/deploy-aws" I manually switched "nodejs8.10" to "nodejs12.x" in line 326. Otherwise AWS refused to accept the lambda.

After that I finally got it working somehow: Lambda is still uploaded to "Virginia", but I can now create german skills with litexa!

As my remaining complaint is adressed in issue #77 I will close this one.
